contracts/Strategy.sol

```diff
@@ -124,33 +156,44 @@ contract Strategy is BaseStrategy {
         }
     }
 
+    // use this during a migration to maintain the strategy's previous boost
+    function manualStakeAsMaxWeighted(
```

Comment:
optional design ideas:
1. could kill access to this method after it's used once
2. if we expect it to be used multiple times, might want to do a `ybs.unstake()` first so that the % parameter is always applied to the full balance. (ofc, this same effect can be had by just calling emergency done via `emergencyUnstake` atomically)